package.json

```diff
@@ -26,25 +26,25 @@
     "express": "^4.16.3",
     "express-prom-bundle": "^5.1.5",
     "google-protobuf": "3.5.0",
```

Comment:
oh, I got confused at first - none of the dependencies actually changed, it's just auto formatting I suppose